.github/workflows/ci.yml

```diff
@@ -106,61 +104,27 @@ jobs:
 
       - name: Srtool build
         id: srtool_build
-        uses: chevdor/srtool-actions@v0.9.2
+        uses: chevdor/srtool-actions@v0.8.0
         with:
-          image: paritytech/srtool
-          tag: 1.77.0
           chain: ${{ matrix.runtime }}
           runtime_dir: runtime
+          profile: "production"
 
-      - name: Summary
```

Comment:
Did this while debugging the srtool-build. It turns out that a downgrade is all that was needed XD.

Regardless, we could simplify here a bit as subwasm is included in srtool.